package.json

```diff
@@ -62,6 +62,8 @@
     "i18next": "^23.1.0",
     "i18next-browser-languagedetector": "^7.0.2",
     "jotai": "^2.6.4",
+    "jotai-effect": "^0.6.0",
```

Comment:
Is jotai-effect still used? Didn't see it anywhere.

Reply:
Yes, it is used for something else. When the fundingStage is no longer in `started` state where we are on qr screen we stop the polling and subscription for funding tx.